source/libs/utils.ts

```diff
@@ -204,3 +196,22 @@ export function getLatestVersionTag(tags: string[]): string {
 
 	return latestVersion;
 }
+
+export function parseRoute(pathname: string) {
+	const [leadingSlash, user, repository, route, ...next] = pathname.split('/');
```

Comment:
The reason I left it was because whenever it is used we need to join it with a slash if we drop it we ould have to do.
'/' + pathparts.

If you want to drop it I would do.

```suggestion
	const [leadingSlash, user, repository, route, ...next] = pathname.split('/').slice(1);
```

Reply:
I just saw https://github.com/sindresorhus/refined-github/pull/3083/files/a68e0c0252099c454e7727e1c0991b63adef8b3d#diff-86e7e1ea0f743a9a248751ef7c7bc4d6 so never mind